pkgs/top-level/all-packages.nix

```diff
@@ -7577,6 +7577,9 @@ with pkgs;
 
   dub = callPackage ../development/tools/build-managers/dub { };
 
+  inherit (callPackage ../build-support/dlang/dub-support.nix { })
+    buildDubPackage dub-to-nix makeDubDep;
```

Comment:
Why not `dub2nix`?

Reply:
The original choice for the name was because this was based off of `nuget-to-nix`, and I forgot to check whether other locker scritps used `-to-` or `2`.
The second reason is that after creating this helper, I found out about the existing `dub2nix` repository, which is an IFD based non-shell application which also does something similar to this. I kept the name as `dub-to-nix` to make the two things distinct.
Although, I am not opposed to changing the name.